openedx/core/djangoapps/xmodule_django/models.py

```diff
@@ -181,6 +183,19 @@ class UsageKeyField(OpaqueKeyField):
     KEY_CLASS = UsageKey
 
 
+class UsageKeyWithRunField(UsageKeyField):
```

Comment:
@ormsbee What are you thoughts on supporting Old Mongo usage-keys (where run disappears) for block structures?  The 3 approaches I see are:
1. Fill in the missing run at time of deserialization - but makes block structures dependent on modulestore for old Mongo courses (approach taken here).
2. Store `CourseKey` along with `UsageKey` in the model, so the model has the information to fill in the run.
3. Don't bother with storing `UsageKey`s.  Instead, just store `CourseKey`s since block-structures are for the entire course at the moment anyway.  However, this is a slightly larger change since the block_structure layer currently deals with usage-keys everywhere - per an earlier decision to support subtrees.